service/hook/github/github.go

```diff
@@ -64,6 +65,7 @@ type PullRequestInfoModel struct {
 	Body           string          `json:"body"`
 	Merged         bool            `json:"merged"`
 	Mergeable      *bool           `json:"mergeable"`
+	Draft          bool            `json:"draft"`
```

Comment:
This could be pointer type, so later when we fill up the model to return we can set the env only when it has true value only. This way we can avoid some spamming by not setting env vars with false value as tue user still can use the var as boolean(set and true or not present). Wdyt?